.changeset/real-carrots-share.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': major
```

Comment:
this shouldn't be `major` if it's going into `master`

Reply:
oh that was me just accepting the first thing changesets offered me, i thought the order was reversed. oops!

Reply:
```suggestion
'@sveltejs/kit': patch
```